src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -440,8 +505,7 @@ static UCS_CLASS_DEFINE_NEW_FUNC(uct_rc_mlx5_iface_t, uct_iface_t, uct_md_h,
 
 static UCS_CLASS_DEFINE_DELETE_FUNC(uct_rc_mlx5_iface_t, uct_iface_t);
 
-static uct_rc_mlx5_iface_ops_t uct_rc_mlx5_iface_ops = {
-    {
+static uct_rc_iface_ops_t uct_rc_mlx5_iface_ops = {
```

Comment:
why is it renamed?

Reply:
removed uct_rc_mlx5_iface_ops_t.iface_tag_init and uct_rc_mlx5_iface_ops_t itself.

Reply:
ok, i see it's fixed in last commit